backend/src/api/endpoints/user.py

```diff
@@ -22,6 +27,8 @@
 from fastapi.security import OAuth2PasswordRequestForm
 from sqlalchemy.orm import Session
 
+# no authentication dependency for the whole group here,
+# as login and register need to be accessible withaut a token
```

Comment:
I'm thinking about moving the login and register route into a separate endpoint file, e.g. `auth.py` or similar. This way we could use the APIRouter in `user.py` to add the token dependency, like in the other endpoint files.

Reply:
great idea! please do it